ruff.toml

```diff
@@ -0,0 +1,5 @@
+line-length = 120
+
+[lint]
+select = ["C", "E", "F", "R", "W", "I", "N", "Q", "UP", "RUF", "ISC", "T20"]
```

Comment:
I'd strongly suggest `B` as well, it catches the lambda capture issue that people get got by often.

Other ones to consider:
- The various pylint ones, `PLC`, `PLE`, `PLW` for conventions, errors, and warnings
- `TC` for some type checking lint rules
- `PERF` for some simple performance checks

Reply:
I'm fine with that, I didn't review all the individual rules under those. Only one I feel strongly about is B